src/puppetlabs/dujour/version_check.clj

```diff
@@ -26,13 +26,15 @@
     map?    ProductCoords))
 
 (def RequestValues
-  {:product-name ProductName
+  {(schema/optional-key :certname) schema/Str
+   (schema/optional-key :cacert) schema/Str
```

Comment:
This is already `host-id` and `site-id` when sent to updates.puppetlabs.com, below. For integrating with this library, however, other services will pass in `certname` and `cacert` for transparency.